cmd/cloud/dbt.go

```diff
@@ -119,19 +111,91 @@ func deployDbt(cmd *cobra.Command, args []string) error {
 
 	// deploy the dbt project as a bundle
 	deployBundleInput := &cloud.DeployBundleInput{
-		BundlePath:   dbtProjectPath,
-		MountPath:    mountPath,
-		DeploymentID: deploymentID,
-		BundleType:   dbtBundleType,
-		Description:  deployDescription,
-		Wait:         waitForDeploy,
-	}
-	err = DeployBundle(deployBundleInput, platformCoreClient, astroCoreClient)
+		BundlePath:         dbtProjectPath,
+		MountPath:          mountPath,
+		DeploymentID:       deploymentID,
+		BundleType:         dbtBundleType,
+		Description:        deployDescription,
+		Wait:               waitForDeploy,
+		PlatformCoreClient: platformCoreClient,
+		CoreClient:         astroCoreClient,
+	}
+	return DeployBundle(deployBundleInput)
+}
+
+func newDbtDeleteCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete DEPLOYMENT-ID",
+		Short: "Delete a dbt project from a Deployment on Astro",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  deleteDbt,
+		Example: `
+Specify the ID of the Deployment on Astronomer you would like to delete this dbt project from:
+
+  $ astro dbt delete <deployment ID>
+
+Menu will be presented if you do not specify a deployment ID:
+
+  $ astro dbt delete
+`,
+	}
+
+	cmd.Flags().StringVarP(&mountPath, "mount-path", "m", "", fmt.Sprintf("Path to mount dbt project in Airflow, for reference by DAGs. Default %s{dbt project name}", dbtDefaultMountPathPrefix))
+	cmd.Flags().StringVarP(&dbtProjectPath, "project-path", "p", "", "Path to the dbt project to delete from the Deployment. Default current directory")
+	cmd.Flags().StringVar(&workspaceID, "workspace-id", "", "Workspace for your Deployment")
+	cmd.Flags().StringVarP(&deploymentName, "deployment-name", "n", "", "Name of the Deployment to deploy to")
+	cmd.Flags().StringVarP(&deployDescription, "description", "", "", "Description to store on the deploy")
```

Comment:
We should update descriptions to reflect this is a delete

Reply:
We're no longer adding unstructured information to the description that is already on the deploy object. The UI should understand that the deploy is a delete and show that to the user.

Reply:
oh sorry, i meant the description of the flag.